ui.tests/test-module/libs/commons/localeDataSets.js

```diff
@@ -15,7 +15,6 @@ const selectors = {
     SUBMIT: '.submit span',
     TELEPHONE_INPUT: 'div[class="cmp-adaptiveform-telephoneinput"] label',
     TEXT_INPUT: '.cmp-adaptiveform-textinput__label',
-    TITLE: '.cmp-title__text',
```

Comment:
Title is removed to match with the master_release650. It is temporarily removed as it was diverged from master_release650 branch.